sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -500,6 +502,24 @@ pi_result _pi_device::initialize() {
   }
   this->ZeComputeQueueGroupIndex = ComputeGroupIndex;
 
+  int CopyGroupIndex = -1;
+  const char *CopyEngine = std::getenv("SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE");
+  bool UseCopyEngine = (!CopyEngine || (std::stoi(CopyEngine) != 0));
+  if (UseCopyEngine) {
+    for (uint32_t i = 0; i < numQueueGroups; i++) {
+      if (((queueProperties[i].flags &
+            ZE_COMMAND_QUEUE_GROUP_PROPERTY_FLAG_COMPUTE) == 0) &&
+          (queueProperties[i].flags &
+           ZE_COMMAND_QUEUE_GROUP_PROPERTY_FLAG_COPY)) {
+        CopyGroupIndex = i;
+        break;
+      }
+    }
+  }
+  if (CopyGroupIndex >= 0)
+    zePrint("NOTE: blitter/copy engine is available\n");
```

Comment:
I think it is even more important to note when it was requested but not found

Reply:
Agreed. Done